docker/rust/Dockerfile

```diff
@@ -1,5 +1,5 @@
-# With this image, only the naive solver will work
-ARG SOLVER_BASE=ubuntu:bionic
+# With this image, only the naive and open solver will work
+ARG SOLVER_BASE=gnosispm/dex-open-solver:v0.0.9-dev
```

Comment:
This is an interesting change. We were using a standard ubuntu image before, but now this is some specialised base image? What are some of the main differences here?

Reply:
The new image contains the open-solver